lib/rebay_api_web/controllers/bid_controller.ex

```diff
@@ -32,13 +32,45 @@ defmodule RebayApiWeb.BidController do
     |> Map.put("uuid", Ecto.UUID.generate())
 
     with {:ok, %Bid{} = bid} <- UserItem.create_bid(bid_params) do
+      handle_auto_bids(item.id)
+
       conn
       |> put_status(:created)
       |> put_resp_header("location", Routes.item_bid_path(conn, :show, item.uuid, bid))
       |> render("show.json", bid: bid)
     end
   end
 
+  defp handle_auto_bids(item_id) do
+    item_id
+    |> UserItem.get_auto_bids_by_item
+    |> run_auto_bids(item_id)
```

Comment:
This may be nitpicky, but I think it makes for better readability to implement pipelines when the same message is being passed through. So passing in `item_id` to a function that returns a collection of bids that get passed to `run_auto_bids` doesn't read as well to me as something more like `UserItem.get_auto_bids_by_item(item_id) |> run_auto_bids(item_id). Such a minor thing here, but I think it becomes more helpful with more complicated pipelines.